python/ynab_client.py

```diff
@@ -65,6 +67,3 @@ def findPreviousTransaction(payee_name):
         .filter(Transaction.imported_payee==payee_name)\
         .order_by(Transaction.date.desc())\
         .first()
-
-# Sync with YNAB
-sync()
```

Comment:
Do you think something like this could work, then?

```python
if __name__ == '__main__:
    ynab_client.sync()
    app.run(host='0.0.0.0', port=settings.port)
```